tasks/main.yml

```diff
@@ -37,15 +37,30 @@
   stat: path=/etc/riak/riak.conf.dist
   register: dist
 
+- name: Check if this is first pass
+  stat: path=/etc/riak/riak_shell.config.dist
+  register: distts
+  when: riak_package == "riak-ts"
```

Comment:
Potentially it could be, but since we aren't providing a method to pull down EE packages within this role, I don't think it matters so much.